cmd/juju/commands/bootstrap_interactive.go

```diff
@@ -41,10 +41,10 @@ func queryCloud(clouds []string, defCloud string, scanner *bufio.Scanner, w io.W
 	// add support for a default (empty) selection.
 	clouds = append(clouds, "")
 
-	verify := interact.MatchOptions(clouds, errors.Errorf("Invalid cloud."))
+	verify := interact.MatchOptions(clouds, "Invalid cloud.")
 
 	query := fmt.Sprintf("Select a cloud [%s]: ", defCloud)
-	cloud, err := interact.QueryVerify([]byte(query), scanner, w, verify)
+	cloud, err := interact.QueryVerify(query, scanner, w, w, verify)
```

Comment:
so, I'm planning to migrate this code very soon to use pollster instead, so I think I'd like to avoid putting a bunch of comments here that will be obviated in my next PR.